app/views/admin/groups/index.html.erb

```diff
@@ -1,5 +1,27 @@
 <% if @event.has_groups? %>
 has Groups
+
+<% @event_groups.each do |event_group| %>
```

Comment:
To understand that this only displays when `<% if @event.has_groups? %>` returns true, we should properly nest this under the condition.